github_connector/models/github_repository_branch.py

```diff
@@ -190,6 +192,18 @@ def _download_code(self):
                         branch._download_code()
         return True
 
+    def get_github_login(self):
+        """Get credentials for private repos."""
+        login = ''
+        if tools.config.get('github_token'):
+            login = "%s@" % (tools.config.get('github_token'))
```

Comment:
According to [docs](https://docs.github.com/en/developers/apps/authenticating-with-github-apps#http-based-git-access-by-an-installation), this should be:

```suggestion
            login = "x-access-token:%s@" % (tools.config.get('github_token'))
```